source/CINetworkMessageManager.h

```diff
@@ -30,8 +30,20 @@ class NetworkMessageManager {
     /** The timestamp of the next message to send. */
     int _timestamp;
     
-    /**Thie id of the player who was won the game. -1 if the game is still ongoing. */
+    /** The id of the player who was won the game. -1 if the game is still ongoing. */
     int _winner_player_id;
+
+    /** Whether or not game has started */
+    int _game_started;
```

Comment:
Instead of using game started and init name sent, can you use the game state enum already present?

Reply:
Also, can we use camelCase for these fields instead of snake_case? And can these be turned into bools or is there a reason that they are ints?

Reply:
I will take a look at the game state enum. `_game_started` is an int because I use 0 to represent before game started, 1 to represent game started is pressed, and 2 to represent after game started so that the game started message only gets sent once. https://github.com/kevinsun-dev/core-impact/pull/73/files#diff-bc50db4ee92732ed90352349030e4dc6b6aabc74131a7e1f579c0035d7b6bf01R270

Reply:
also, in regards to camelCase, I was following the other fields... do we want to change all fields to snake_case?

Reply:
Most class fields in our code have started with an underscore and then been camelCase, so if we could follow that convention in NetworkMessageManager that would be great

Reply:
If you're going to use an int field like this you should make it explicit what each int value is for in the comments. But i'd prefer using an enum for this